run_test.sh

```diff
@@ -49,17 +49,14 @@ ln -s ${tmp_dir} test_dir
 cd test_dir
 echo "cd $PWD"
 
-# Clone PICSAR, AMReX and warpx-data
+# Clone AMReX and warpx-data
 git clone https://github.com/AMReX-Codes/amrex.git
 cd amrex && git checkout --detach 88b868099018c191e7d3ea424d3e3420b826dd69 && cd -
-# Use QED brach for QED tests
-git clone https://github.com/ECP-WarpX/picsar.git
-cd picsar && git checkout --detach 8ff12fbf118b9aba7cfe602cb1a5e6da32bf7eef && cd -
 # warpx-data contains various required data sets
 git clone --depth 1 https://github.com/ECP-WarpX/warpx-data.git
 
 # Clone the AMReX regression test utility
-git clone https://github.com/ECP-WarpX/regression_testing.git
+git clone https://github.com/AMReX-Codes/regression_testing.git
```

Comment:
Temporary until https://github.com/AMReX-Codes/regression_testing/pull/99 is merged:
```suggestion
git clone --branch fix-getRecentFilename https://github.com/ax3l/regression_testing.git
```